src/asm/rgbasm.5

```diff
@@ -35,13 +37,35 @@ There are two syntaxes for comments. In both cases, a comment ends at the end of
 the line. The most common one is: anything that follows a semicolon
 \[dq]\&;\[dq] (that isn't inside a string) is a comment. There is another
 format: anything that follows a \[dq]*\[dq] that is placed right at the start of
-a line is a comment.
+a line is a comment. The assembler removes all comments from the code before
+doing anything else.
+.Pp
+Sometimes lines can be too long and it may be needed to split them. The syntax
```

Comment:
I'd say "it may be necessary to split them", personally, but I'm splitting hairs here!